pkg/operator/ceph/nfs/nfs.go

```diff
@@ -291,20 +291,21 @@ func validateGanesha(context *clusterd.Context, clusterInfo *cephclient.ClusterI
 }
 
 // create and enable default RADOS pool
-func (r *ReconcileCephNFS) createDefaultNFSRADOSPool(n *cephv1.CephNFS) error {
+func (r *ReconcileCephNFS) configureNFSPool(n *cephv1.CephNFS) error {
 	poolName := n.Spec.RADOS.Pool
+	logger.Infof("configuring pool %q for nfs", poolName)
 
-	args := []string{"osd", "pool", "create", poolName}
-	output, err := cephclient.NewCephCommand(r.context, r.clusterInfo, args).Run()
+	_, err := cephclient.GetPoolDetails(r.context, r.clusterInfo, poolName)
 	if err != nil {
-		return errors.Wrapf(err, "failed to create default NFS pool %q. %s", poolName, string(output))
+		return errors.Wrapf(err, "pool %q could not be retrieved", poolName)
```

Comment:
Maybe we could improve this error message to instruct users to create the `.nfs` pool via `CephBlockPool` CRD? Maybe pointing them to a section in the NFS doc about that? (which I think we'll also have to add?)